src/applications/disability-benefits/526EZ/tests/components/ConfirmationPoll.unit.spec.jsx

```diff
@@ -75,7 +75,7 @@ describe('ConfirmationPoll', () => {
     ]);
     // TODO: Figure out why this is causing an error in the console even though the test passes
     //  It may have something to do with unmounting the component before a `setState` goes through
-    mount(<ConfirmationPoll pollRate={10} />);
+    shallow(<ConfirmationPoll pollRate={10} />);
```

Comment:
This was resulting in `AssertionError: expected 2 to equal 3`, using `shallow` instead resolved the issue.